### PR TITLE
remove apis being depricated in k8s 1.22

### PR DESCRIPTION
**Pull request by ashleyschuett**
Signed-off-by: Ashley Schuett <aschuett@redhat.com>

**What this PR does / why we need it**:

These APIs are being removed in k8s 1.22. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Update v1beta1 apis to v1
```

### CODE REVIEW

**Comment by ashleyschuett** [comment]
/hold 

blocked by https://github.com/kubevirt/kubevirtci/pull/640

**Review by davidvossel — APPROVED** [review]
/approve

**Comment by rmohr** [comment]
/lgtm
/unhold
/retest

Tests seem to be fine now.

File: cluster-up/cluster/kind/manifests/kube-calico.yaml.in [hunk 1]
```diff
@@ -54,7 +54,7 @@ data:
 ---
 # Source: calico/templates/kdd-crds.yaml
 
-apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (rmohr) [inline cluster-up/cluster/kind/manifests/kube-calico.yaml.in:57]: Oh this file is coming from `kubevirt/kubevirtci` and has to be changed there.

**Review by rmohr — CHANGES_REQUESTED** [review]
/lgtm cancel

**Comment by ashleyschuett** [comment]
closing due to merge of https://github.com/kubevirt/kubevirtci/pull/644

**Pull request closed without merging**